src/include/platform/NetworkCommissioning.h

```diff
@@ -79,8 +83,12 @@ class Iterator
 struct Network
 {
     uint8_t networkID[kMaxNetworkIDLen];
-    uint8_t networkIDLen;
-    bool connected;
+    uint8_t networkIDLen = 0;
+    // #if CHIP_DEVICE_CONFIG_ENABLE_WIFI_PDC
```

Comment:
Why is that commented out?  It should either be here or not be here...

Reply:
hm yeah must have messed something up during one of my rebases.